acceptance_tests/__init__.py

```diff
@@ -20,14 +20,18 @@ def web_driver_connection_error(e):
 
 @retry(retry_on_exception=web_driver_connection_error, wait_fixed=1000, stop_max_attempt_number=30)
 def create_browser():
-
-    if os.getenv('HEADLESS', 'True') == 'True':
-        chromedriver_binary.add_chromedriver_to_path()
-        chrome_options = webdriver.ChromeOptions()
-        chrome_options.add_argument("--no-sandbox")
-        return Browser('chrome', headless=True, options=chrome_options)
+    driver_type = os.getenv('WEBDRIVER', 'chrome')
+    headless = os.getenv('HEADLESS', 'True') == 'True'
+    if driver_type.lower() == 'firefox':
+        return Browser('firefox', headless=headless)
     else:
-        return Browser('chrome')
+        if headless:
+            chromedriver_binary.add_chromedriver_to_path()
+            chrome_options = webdriver.ChromeOptions()
+            chrome_options.add_argument("--no-sandbox")
+            return Browser('chrome', headless=True, options=chrome_options)
```

Comment:
Not sure why the firefox one passes `headless=headless` but this passes `headless=True`; headless will always be True here

Reply:
Its the different ways chrome and firefox are configured

Reply:
That shouldn't stop us passing in `headless=headless` rather than `headless=True` if the variable is guaranteed to be `True`